code/modules/mob/living/carbon/human/human.dm

```diff
@@ -23,6 +23,7 @@
 	. = ..()
 
 	RegisterSignal(src, COMSIG_COMPONENT_CLEAN_FACE_ACT, .proc/clean_face)
+	RegisterSignal(src, COMSIG_HUMAN_INSANE, .proc/Spread_Panic)
```

Comment:
There is no need to do this, just call that proc in death()

Reply:
You do realize that is for spreading panic when someone else panics, right? Which is a function of the base game and is what gets replicated here?

Reply:
I think I understand what you mean, and that's to remove the code in death(gibbed) and just have death(gibbed) call Spread_Panic() which I've done